### PR TITLE
fix(#419) preserve body padding-right in removeScroll

### PR DESCRIPTION
**Pull request by Tyneor**
#419 
🍁🚬

### CODE REVIEW

**Comment by Tyneor** [comment]
Whoops I forgot to take into account the scrollbar being on the left.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # @melt-ui/svelte
 
+## 0.37.4
```
Comment (TGlide) [inline CHANGELOG.md:3]: Why was this modified? Did you run npx changeset version on accident?

File: src/docs/content/controlled.md [hunk 2]
```diff
@@ -157,7 +157,9 @@ const customMeltEvent: MeltEvent<typeof originalEvent> = new CustomEvent(
 )
 ```
 
-By marking the custom event as `cancelable`, you may use `preventDefault` on it, which signals Melt to immediately stop handling the event, thus preventing the builder's default behavior from happening.
+By marking the custom event as `cancelable`, you may use `preventDefault` on it, which signals Melt
+to immediately stop handling the event, thus preventing the builder's default behavior from
+happening.
```
Comment (TGlide) [inline src/docs/content/controlled.md:162]: Why was this modified?

File: src/docs/previews/popover/main/tailwind/index.svelte [hunk 3]
```diff
@@ -6,7 +6,9 @@
 	const {
 		elements: { trigger, content, arrow, close },
 		states: { open },
-	} = createPopover();
+	} = createPopover({
+		forceVisible: true,
+	});
```
Comment (TGlide) [inline src/docs/previews/popover/main/tailwind/index.svelte:11]: Same here

Comment (TGlide) [reply src/docs/previews/popover/main/tailwind/index.svelte:11]: EDIT: I think the git history is a bit weird for whatever reason 🤔

Comment (Tyneor) [reply src/docs/previews/popover/main/tailwind/index.svelte]: Indeed, I should have followed your contributing guide like I did for my first pull request 😅(great guide btw)

I tried resetting my fork but it auto-closed this PR and I couldn't get it to work anyway so I created a new PR with just one commit this time.

Sorry for the mess !

Comment (TGlide) [reply src/docs/previews/popover/main/tailwind/index.svelte]: No worries, thank you for the hard work!!
